lms/djangoapps/courseware/rules.py

```diff
@@ -68,13 +68,13 @@ def compare(self, control, candidate):
 
     def publish(self, result):
         if not result.match:
-
-            LOG.warning(
-                u"StaffAccessExperiment: control=%r, candidate=%r\n%s",
-                result.control,
-                result.candidates[0],
-                "".join(traceback.format_stack(limit=10))
-            )
+            pass
```

Comment:
Why was this commented? Shouldn't we continue to log this warning?

Reply:
@davestgermain should this be put back in?

Reply:
Nice catch. I think there's a python3 incompatibility somewhere in `laboratory`, though. If so, it'll surface again.